src/components/shared/WipErr/WipErr.js

```diff
@@ -5,17 +5,29 @@ import * as shapes from '../../shapes';
 import { formatError } from '../../../helpers/utils';
 import { ErrorSplash, FrownSplash, LoadingSplash } from '../Splash/Splash';
 
+const errorStatusNotFound = err => (err.response && err.response.status === 404);
```

Comment:
err?.response?.status === 404

Reply:
It was "old style". If you're already here and you're changing **everything** to be "new style", please, change it too...

Reply:
interesting that eslint misses it... 
but found a plugin that we may add for this: https://www.npmjs.com/package/eslint-plugin-prefer-optional-chaining
what do you think?